lib/livekit/auth_mixin.rb

```diff
@@ -3,10 +3,16 @@
 module LiveKit
   # Create authenticated headers when keys are provided
   module AuthMixin
-    def auth_header(video_grant)
+    def auth_header(
+      video_grant,
+      sip_grant: nil
+    )
       headers = {}
       t = ::LiveKit::AccessToken.new(api_key: @api_key, api_secret: @api_secret)
       t.add_grant(video_grant)
```

Comment:
should we handle passing in a nil video grant?

Reply:
It does work right now by constructing a default video grant but I can handle the case explicitly, yes